include/RevCore.h

```diff
@@ -316,6 +323,8 @@ class RevCore {
 
   std::bitset<_MAX_HARTS_> CoProcStallReq{};
 
+  uint64_t cycles{};  ///< RevCore: The number of cycles executed
+
```

Comment:
The RISC-V spec says that `rdcycle` should return the number of cycles executed by the _core_ so far, not the hart, which is why this is in `RevCore` instead of `RevRegFile`.